client/blocks/reader-subscription-list-item/connected.jsx

```diff
@@ -65,30 +67,34 @@ class ConnectedSubscriptionListItem extends React.Component {
 			siteId,
 			showNotificationSettings,
 			showLastUpdatedDate,
+			isEmailBlocked,
 			isFollowing,
 			followSource,
 			railcar,
 		} = this.props;
-		const isEmailBlocked = userSettings.getSetting( 'subscription_delivery_email_blocked' );
 
 		return (
-			<SubscriptionListItem
-				translate={ translate }
-				feedId={ feedId }
-				siteId={ siteId }
-				site={ site }
-				feed={ feed }
-				url={ url }
-				showNotificationSettings={ showNotificationSettings && ! isEmailBlocked }
-				showLastUpdatedDate={ showLastUpdatedDate }
-				isFollowing={ isFollowing }
-				followSource={ followSource }
-				railcar={ railcar }
-			/>
+			<div>
```

Comment:
Can we use `Fragment` here instead to avoid adding an extraneous div?

Reply:
Thank you @bluefuton! Didn't know there is this component. Updated the PR with this fix in 6f54f0eefb98af2c54062e23ed852ea6f8508cfe.